.github/workflows/release.yml

```diff
@@ -45,5 +45,7 @@ jobs:
           rm -fv minisign.key
   upload:
     environment: production
+    if: github.repository == 'bufbuild/plugins'
+    runs-on: ubuntu-22.04
```

Comment:
Hoping this will resolve https://github.com/bufbuild/plugins/actions/runs/4020280875.